shared/pages/PartialClosure/PartialClosure.js

```diff
@@ -81,15 +76,14 @@ export default class PartialClosure extends Component {
     }
   }
 
-  constructor({ tokensData, currenciesData, match: { params: { buy, sell, locale } }, history, ...props }) {
+  constructor({ tokensData, currenciesData, match: { params: { buy, sell } }, intl: { locale }, history, ...props }) {
     super()
 
     const sellToken = sell || ((!isWidgetBuild) ? 'eth' : 'btc')
     const buyToken = buy || ((!isWidgetBuild) ? 'btc' : config.erc20token)
-    const localization = locale ? `/${locale}` : ''
 
     if (!(buy && sell) && !props.location.hash.includes('#widget')) {
-      history.push(`${localization}/exchange/${sellToken}-to-${buyToken}`)
+      this.props.history.push(localisedUrl(locale, `/exchange/${sellToken}-to-${buyToken}`))
```

Comment:
```suggestion
      history.push(localisedUrl(locale, `/exchange/${sellToken}-to-${buyToken}`))
```

![image](https://user-images.githubusercontent.com/24671211/52650952-55f0ee00-2efc-11e9-89d4-0ca07e07b75f.png)